golang/cosmos/x/swingset/client/cli/tx.go

```diff
@@ -91,7 +92,7 @@ func GetCmdDeliver() *cobra.Command {
 // InstallBundle message in a transaction.
 func GetCmdInstallBundle() *cobra.Command {
 	cmd := &cobra.Command{
-		Use:  "install-bundle <JSON>/@<FILE>/-",
+		Use:  "install-bundle [--compress=true] <JSON>/@<FILE>/-",
```

Comment:
Hadn't we agreed to make `--compress=true` the default?  I'd prefer that, since we're trying to phase out chains that don't support compression.

Reply:
Oh? I thought we were going to make it opt-in until Vaults+1. No matter - I'll just change the default and leave it off of the synopsis.